docs/tutorial/README.md

```diff
@@ -0,0 +1,121 @@
+
+https://github.com/dapr/quickstarts/tree/master/tutorials/hello-kubernetes
+
+Instead of deploying Dapr as a sidecar we are going to use Dapr Shared instances. 
+
+
+## Prerequisites and Installation
+
+Before proceeding make sure that you have the necessary tools installed on your system. We will create a local KinD cluster to install Dapr, some applications, and an instance of Dapr Shared.
+
+To get started, make sure you have the following CLIs installed:
+
+- [Docker](https://www.docker.com/)
+
+- [KinD (Kubernetes in Docker)](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+- [Helm](https://helm.sh/docs/intro/install/)
+
+
+## Creating a local Kubernetes cluster with KinD: 
+
+Create a Kubernetes cluster with KinD defaults by running the following command:
+
+```bash
+kind create cluster
+```
+
+Next install a version of the Dapr control plane into the cluster:
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+helm upgrade --install dapr dapr/dapr \
+--version=1.13.2 \
+--namespace dapr-system \
+--create-namespace \
+--wait
+```
+Or use the Dapr CLI to install the latest version with the following command
+
+`dapr init -k`
+## Running the Hello Kubernetes example
+
+Next install a Redis instance into the cluster using Helm
+
+```shell
+helm install redis oci://registry-1.docker.io/bitnamicharts/redis --version 17.11.3 --set "architecture=standalone" --set "master.persistence.size=1Gi"
+```
+
+Once Redis is installed you can deploy our application workloads, including a statestore component by running: 
+
+```shell
+kubectl apply -f deploy/
+```
+
+This creates a statestore component, a Node application and a Python application. 
+
+If you inspect the `deploy/node.yaml` and `deploy/python.yaml` files you see that both are define two environment variables: 
+
+```yaml
+        - name: DAPR_HTTP_ENDPOINT
+          value: http://nodeapp-dapr.default.svc.cluster.local:3500
+        - name: DAPR_GRPC_ENDPOINT
+          value: http://nodeapp-dapr.default.svc.cluster.local:50001
+```
+
+These two environment variables let the Dapr SDK know where the Dapr endpoints are hosted (usually for the sidecar these are located on `localhost`).
+
+Because these workloads are not annotated with Dapr annotations, the Dapr Control Plane will not inject a Dapr sidecar, instead we will create two instances of the Dapr runtime using Dapr Shared for our services to use.
+
+
+## Creating two Dapr Shared instances for our services
+
+For each application service that needs to talk to the Dapr APIs we need to deploy a new Dapr Shared instance. Each instance have a one to one relationship with Dapr Application Ids. 
+
+Let's create a new Dapr Shared instance for the Node (`nodeapp`) application: 
+
+```sh
+helm install nodeapp-shared oci://docker.io/daprio/dapr-shared-chart --set shared.appId=nodeapp
```

Comment:
I feel like the helm chart here abstracts what is actually happening too much. Can you add a link to the helm chart at least so that users can inspect the values?